MultiPage-Clone/index.html

```diff
@@ -1,19 +1,325 @@
+                  ></path>
+                </g>
+              </svg>
+              <span>2035</span>
+            </div>
+            <h1 class="heading">PRICKLES & CO</h1>
+            <p class="hero-card-p">BRING NATURE INDOORS</p>
+            <button class="hero-btn">Shop Now</button>
+          </div>
+        </section>
+        <section class="container">
+          <section class="card-container">
+            <div class="card card-1">
+              <h2 class="card-heading">CACTI</h2>
+              <button class="hero-btn">Shop Collection</button>
+            </div>
+            <div class="card card-2">
+              <h2 class="card-heading">PLANTS</h2>
+              <button class="hero-btn">Shop Collection</button>
+            </div>
+            <div class="card card-3">
+              <h2 class="card-heading">SUCCULENTS</h2>
+              <button class="hero-btn">Shop Collection</button>
+            </div>
+          </section>
+          <section class="columns">
+            <section class="about-us" id="about">
+              <div class="about-us-card">
+                <h2 class="about-us-heading">About US</h2>
+                <p class="about-us-p">
+                  Welcome to Prickles & Co.! We're not just a regular shop; 
+                  we're all about plants and making your home greener.
+                   We've been around since 2035, and we're here to bring nature inside.
+                    Our store is filled with awesome cacti, plants, and succulents to spruce up your space. 
+                    <br />
+                  <br /><br />
 
+                  You might have some questions, and that's cool – check out our FAQs. And if you need to reach us, just hit "Contact." 
+                  The best part? We'll even deliver these lovely plants to your doorstep. So, come on in, explore, and let's have a chat about how 
+                  Prickles & Co. can make your home more lush and vibrant.
+                </p>
+              </div>
+              <div class="/about-us-img-1"></div>
```

Comment:
Try to use sematic html. It's important to use <img> tags for images because they have at attributes, so if anyone is blind, the computer knows to read it out to them. A div does not provide the same use or tell the user much